ElixrTalentConnect/ElixrTalentConnect/Login/LoginViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  LoginViewController.swift
+//  ElixrTalentConnect
+//
+//  Created by Shemil Tom on 10/01/24.
+//
+
+import UIKit
+import LocalAuthentication
+
+class LoginViewController: UIViewController, UITextFieldDelegate  {
+    
+    @IBOutlet weak var elixrSymbolImage: UIImageView!
+    @IBOutlet weak var signinBottomView: UIView!
+    
+    @IBOutlet weak var userEmail: UITextField!
+    @IBOutlet weak var userPassword: UITextField!
+    
+    private var viewModel = LoginViewModel()
+    
+    @IBAction func signInTapped(_ sender: Any) {
+        let loginModel = LoginModel(email: userEmail.text ?? "", password: userPassword.text ?? "")
+        let validationResult = viewModel.validateCredentials(model: loginModel)
+        
+        if validationResult.isValid {
+            viewModel.authenticateWithBiometrics { [weak self] (success, error) in
+                if success {
+                    self?.performSegue(withIdentifier: "LoginToHome", sender: nil)
+                } else {
+                    if let error = error {
+                        self?.showAlert(message: "Biometric authentication failed: \(error.localizedDescription)")
+                    }
+                }
+            }
+        } else {
+            showAlert(message: validationResult.message ?? "Invalid credentials.")
+        }
+    }
+    
+    
+    @IBAction func GoTOSignUpTapped(_ sender: Any) {
+        performSegue(withIdentifier: "SignUpSegue", sender: self)
+    }
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        elixrSymbolImage.layer.cornerRadius = 20
```

Comment:
Can we have separate methods to handle the bottom view setup instead of keeping everything in the view model?